install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,209 @@
+[[install-config-configuring-nsx-t-sdn]]
+[%hardbreaks]
+= Configuring NSX-T SDN
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[nsx-t-sdn-and-openshift]]
+== NSX-T SDN and {product-title}
+
+VMware NSX-T provides advanced software-defined networking (SDN), security and visibility
+to container environments that simplifies IT operations and extends {product-title}’s native
+networking capabilities.
+
+NSX-T supports Virtual Machine, Bare-metal, and container workloads across multiple clusters. This allows
+organisations to have complete visibility using a single SDN across the entire environment.
+
+
+See xref:../architecture/networking/network_plugins.adoc#nsx-sdn[Networking]
+for more information on how NSX-T is integrated with {product-title}.
+
+[[nsx-t-sdn-operations-workflow]]
+== Example Topology
+
+There is a *T0 router* on the top that connects physical with virtual world. We also have *T1 router* acting as a default gateway for the OpenShift VMs.
+
+Those VMs have *two vNICs* each. One vNIC is connected to Management Logical Switch for accessing the VMs. The second vNIC is connected to a dump Logical Switch and is used by nsx-node-agent to uplink the POD networking. For further details, we strongly suggest reading the link:https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.3/nsxt_23_ncp_OpenShift.pdf[NSX Container Plug-in for OpenShift].
```

Comment:
@RobbieJVMW  I get a 404 at that link. Can you confirm the link?

Reply:
The link should be : https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.4/nsxt_24_ncp_openshift.pdf

If you could check it works for you, its fine for me.